ColorPatterns/Controller/ViewController.swift

```diff
@@ -10,35 +10,36 @@ import UIKit
 
 class ViewController: UIViewController {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/134e78896b7d06b391bc723303fb96a8b747df75/ColorPatterns/Controller/ViewController.swift#L11"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>